Sources/Onboarding/Styles/DaxDialogStyles.swift

```diff
@@ -81,10 +105,12 @@ extension OnboardingStyles {
 }
 
 extension Color {
-    static let blue70 = Color(0x1E42A4)
```

Comment:
![Screenshot 2024-10-14 at 4 33 36 PM](https://github.com/user-attachments/assets/7633be60-fe2d-4214-8cd0-1f687cd63b1a)

It seems that the hex color is the same between light and dark mode as per screenshot but we used different code.

Reply:
nevermind this is only ListButton